content/pages/privacy.mdx

```diff
@@ -0,0 +1,206 @@
+
+* such objection or withdrawal of consent could mean that SSW are unable to provide our services to you, and could unduly prevent us from legitimately providing our services to other customers/clients subject to appropriate confidentiality protections; and
+* even after you have chosen to withdraw your consent, SSW may be able to continue to keep and process your personal information to the extent required or otherwise permitted by law, in particular:
+* to pursue our legitimate interests in a way that might reasonably be expected as part of running our business and which does not materially impact on your rights, freedoms or interests; and
+* in exercising and defending our legal rights and meeting our legal and regulatory obligations.
+
+#### Storage and processing by third parties
+
+Data that SSW collect about you may be stored or otherwise processed by third party services with data centres based outside the EU, such as Google Analytics, Microsoft Azure, Amazon Web Services, Apple, etc and online relationship management tools.  SSW consider that the collection and such processing of this information is necessary to pursue our legitimate interests in a way that might reasonably be expected (eg, to analyse how our customers/clients use our services, develop our services and grow our business) and which does not materially impact your rights, freedom or interests.
+SSW requires that all third parties that act as “data processors” for us provide sufficient guarantees and implement appropriate technical and organisational measures to secure your data, only process personal data for specified purposes and have committed themselves to confidentiality.
+
+#### Duration of retention of your data
+
+SSW will only keep your data for as long as is necessary for the purpose for which it was collected, subject to satisfying any legal, accounting or reporting requirements.  At the end of any retention period, your data will either be deleted completely or anonymised (for example, by aggregation with other data so that it can be used in a non-identifiable way for statistical analysis and business planning).  In some circumstances, you can ask us to delete your data.
+
+#### Keeping your information up-to-date
+
+To ensure that your personal information is accurate and up to date, please promptly advise us of any changes to your information by contacting our data protection officer by email or by post.
+
+### Changes to this Privacy Policy
+
+From time to time, it may be necessary for us to revise this Privacy Policy. Any changes will be in accordance with any applicable requirements under the Privacy Act and the Australian Privacy Principles. SSW may notify you about changes to this Privacy Policy by posting an updated version on this site.
+
+***
+
+If you require any further information about the Privacy Act and the Australian Privacy Principles, you can visit the Federal Privacy Commissioner’s Website (see [www.privacy.gov.au](www.privacy.gov.au)).
```

Comment:
Have we linked to the correct page? SSL has expired on `https://www.privacy.gov.au`

![image](https://github.com/SSWConsulting/SSW.Website/assets/35659961/f6da51e2-12e3-46c7-aff5-7b433e210c86)

Reply:
Maybe its supposed to be this page 

https://www.oaic.gov.au/privacy

Reply:
Fixed